rockspecs/penlight-1.6.0-1.rockspec

```diff
@@ -2,13 +2,13 @@ package = "penlight"
 version = "1.6.0-1"
 
 source = {
-  url = "git://github.com/Tieske/Penlight.git",
+  url = "git://github.com/lunarmodules/Penlight.git",
```

Comment:
we shouldn't be updating existing rockspecs

Reply:
The Lua rocks setup is so eccentric sometimes the official best practice is not always the most practical thing. In this case the orthodox thing to do would be to bump the rockrel in order to update these URLs so that people could still install the old versions. However doing that would actually *break* some workflows. The one in particular I'm thinking of is `luarocks install <full_url_to_rockspec>`. It should be possible to do this against the current repository (since the commits and checksums of what you get when you checkout the 1.6.0 or whatever tag hasn't changed) and get the old version cleanly, but right now that only works because of GitHub's redirects. Given that GitHub redirects are likely to last quite a while in this case it's one we can probably let slide, I'm just pointing out that there are a handful of cases where updating old rockspecs can be the only way to not break external workflows.